packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```diff
@@ -280,16 +281,22 @@ export class GutenbergEditorPage {
 	}
 
 	/**
-	 * Opens the settings sidebar.
-	 *
-	 * @returns {Promise<void>} No return value.
+	 * @returns Whether the Settings sidebar is open or not.
 	 */
-	async openSettings(): Promise< void > {
+	async isSettingsSidebarOpen(): Promise< boolean > {
```

Comment:
Perhaps this should be `private`?

Reply:
Not sure, it looks like something that could be useful outside but I don't have a strong opinion here. Is there anything more to hiding methods like this than to prevent misusage?

Reply:
I don't feel strongly either; by convention I've usually hid these sorts of methods since the caller should not need to call this manually unless they really wanted to check.